pkgs/tools/misc/hyfetch/default.nix

```diff
@@ -0,0 +1,36 @@
+{ python3Packages, lib, fetchFromGitHub }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "HyFetch";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    repo = "hyfetch";
+    owner = "hykilpikonna";
+    rev = "92623417f90f0cf006c0dd2adcf3f24f4308fe0c";
+    sha256 = "sha256-26L2qt+RarRf3+L6+mMy/ZJNVBVirKs5oEclEsImtC0=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    typing-extensions
+    setuptools
+  ];
+
+  doCheck = false;
```

Comment:
The tests pass if you remove the numpy-importing one. Not a huge deal in this case, but it's possible to correct if you want.

Reply:
I added `preCheck` to remove the file, thank you for reminding me.